CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.3
```

Comment:
`## 3.0.3-dev` here and in `pubspec.yaml`. 

(We are following https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change )